odoo/addons/base/models/res_company.py

```diff
@@ -182,6 +183,46 @@ def _onchange_parent_id(self):
                 if self[fname] != self.parent_id[fname]:
                     self[fname] = self.parent_id[fname]
 
+    @api.depends('country_id')
+    def _compute_uninstalled_l10n_module_ids(self):
+        # This will only compute uninstalled modules with auto-install without recursion,
+        # the rest will eventually be handled by `button_install`
+        self.env['ir.module.module'].flush_model(['auto_install', 'country_ids', 'dependencies_id'])
+        self.env['ir.module.module.dependency'].flush_model()
+        self.env.cr.execute("""
+            SELECT country.id,
+                   ARRAY_AGG(module.id)
+              FROM ir_module_module module,
+                   res_country country
```

Comment:
Stick to SQL-92 ?

Reply:
Yes, I'm using the `DATE` type wherever I can in this query :+1: